source/jormungandr/jormungandr/interfaces/v1/LineReports.py

```diff
@@ -92,6 +92,8 @@ def __init__(self):
                                 help="use disruptions valid after this date")
         parser_get.add_argument("until", type=DateTimeFormat(),
                                 help="use disruptions valid before this date")
+        parser_get.add_argument("tags[]", type=six.text_type, action="append",
+                                help="If filled, will restrained the search within the given disruption tags")
```

Comment:
i'd better precise it's a "OR" condition

Reply:
When we use a parameter with multiple values, it's obviously 'or' condition non?